### PR TITLE
Create xdelta.json

### PR DESCRIPTION
**Pull request by vaguiners**
open-source binary diff, differential compression tools, VCDIFF (RFC 3284) delta compression

### CODE REVIEW

**Comment by Ash258** [comment]
Could be simplified to: (No need for preinstall)
`"url": "https://github.com/jmacd/xdelta-gpl/releases/download/v3.1.0/xdelta3-3.1.0-x86_64.exe.zip#/xdelta.exe",`

**Comment by vaguiners** [comment]
The filename include version

________________________________
De: Jakub Čábera <notifications@github.com>
Enviado: sexta-feira, 3 de agosto de 2018 17:31:22
Para: lukesampson/scoop-extras
Cc: vaguiners; Author
Assunto: Re: [lukesampson/scoop-extras] Create xdelta.json (#1095)


Could be simplified to: (No need for preinstall)
"url": "https://github.com/jmacd/xdelta-gpl/releases/download/v3.1.0/xdelta3-3.1.0-x86_64.exe.zip#/xdelta.exe",

File: xdelta.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "license": "GPL-3.0",
     "bin": "xdelta.exe",
     "pre_install": "Get-ChildItem \"$dir\\xdelta3-*.exe\" | Rename-Item -NewName \"$dir\\xdelta.exe\"",
+    "checkver": "github": "https://github.com/jmacd/xdelta-gpl",
```
Comment (rasa) [inline xdelta.json]: This needs to be:
```
 "checkver": {"github": "https://github.com/jmacd/xdelta-gpl"},

**Comment by vaguiners** [comment]
done
another thing, i think this deserves to be on main scoop repo

**Comment by r15ch13** [comment]
No, this would only work when it was an exe. But it has to be extracte first. Your post_install script it the right way to do it. :+1:

**Comment by rasa** [comment]
If it's a command-line only tool, please open a PR in the scoop bucket instead.

**Pull request closed without merging**